tests/unittests/StatementTests.cpp

```diff
@@ -268,6 +268,25 @@ endmodule
     NO_COMPILATION_ERRORS;
 }
 
+TEST_CASE("Basic Case statements") {
```

Comment:
This isn't really testing basic case statements, it's testing the lexing of comments next to the :/ operator. I think I'd just put a more targeted test into LexerTests instead so that it's clear what we're actually trying to catch here.